tools/functional-tester/etcd-runner/main.go

```diff
@@ -212,6 +219,136 @@ func runRacer(eps []string, round int) {
 	doRounds(rcs, round)
 }
 
+func runWatcher(eps []string) {
+	runningTime := 30 * time.Second // time for which operation should be performed
+	noOfPrefixes := 36              // total number of prefixes which will be watched upon
+	watchPerPrefix := 10            // number of watchers per prefix
+	reqRate := 20                   // put request per second
+	keyPrePrefix := 100000          // max number of keyPrePrefixs for put operation
```

Comment:
sure will figure out a more reasonable value. May be keep it just slightly over the value it comes to.